benchmark/README.md

```diff
@@ -2,7 +2,37 @@ Benchmarking
 ===
 
 # Generate Data
-> *TODO* instructions to be provided
+1. Download the available dbgen tool from TPC org: http://www.tpc.org/tpc_documents_current_versions/current_specifications.asp
+There are many different Benchmarks, you just need to download the corresponding tool for TPC-H. Click on Download TPC-H_Tools_[version].zip
+Follow the instructions for registration and download the tool and copy to local disk with free space larger than 300GB.
+
+2. Build the dbgen tool.
+Decompress the zip file, then navigate to “dbgen” folder.
+In the README, it has the instruction on how to build the tool in Linux.
+For Windows users, you can generate the dbgen.exe using Visual Studio: 
+(a)	In the “dbgen” folder, you will see “tpch.sln”, open it using Visual Studio (VS2015 or VS2017 should work).
+(b)	Click on tab “Build”, then “Build Solution”. 
+(c)	Once the build is successful, it should generate "dbgen.exe" and "qgen.exe " in the “Debug” folder.
+
+3. Generate the data.
+In the README, it has the instructions on how to generate a database population in Linux OS.
+For Windows users, you can:
+(a)	Copy “dbgen.exe” from “Debug” folder to the “dbgen” folder. 
+(b)	Open cmd terminal, navigate to “dbgen” folder, and type command: dbgen -vf -s 300, which is generating 300GB TPCH dataset. It could take up to 40 hours to complete. Note that you can use command: dbgen -h, to check all the options. 
+After database population generation is completed, you will see 8 tables with .tbl format:

```

Comment:
The following files will be created:
- customer.tbl
- lineitem.tbl
- nation.tbl
- orders.tbl
- part.tbl
- partsupp.tbl
- region.tbl
- supplier.tbl